scss/partials/_base.scss

```diff
@@ -87,7 +92,8 @@ h5,
 }
 
 h6,
-.h6 {
+.h6,
+.h6--alt .h6--quote {
```

Comment:
Just making sure this is meant to be a nested selector here. I think there might need to be a comma between the classes. If you wanted to simplify this selector and the one's above, you might get away with doing something more like `[class^=h6]` instead (adjusting the number for each level accordingly).

Reply:
Thank you. I totally missed that. applied your approach.